docs/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ title: Changelog
 
 ## main
 
+## 2.53.0
```

Comment:
Can you remove this heading? It will be added as part of the release process.

Reply:
Done!